Healthy/Classes/Extensions/UIView+Helpers.swift

```diff
@@ -31,6 +31,24 @@ extension UIView {
         addSubview(contentView)
     }
 
+    func roundImage() {
+        self.layer.cornerRadius = self.frame.size.width / 2
+        self.clipsToBounds = true
+    }
+
+    func setRadius(radius: Int) {
+        self.layer.cornerRadius = CGFloat(radius)
+    }
+
```

Comment:
```suggestion
```

Please check `cornerRadius` property that exists at the end of the extension. Please use it.